spec/unit/prepare.spec.js

```diff
@@ -783,6 +783,20 @@ describe('prepare', () => {
             Api = rewire('../../bin/templates/cordova/Api');
             prepare = rewire('../../bin/templates/cordova/lib/prepare');
 
+            prepare.__set__('require.resolve', (file) => {
+                if (file === 'cordova-android/framework/defaults.json') {
+                    return path.resolve('./framework/defaults.json');
+                }
+                return path.resolve(file);
+            });
```

Comment:
There's more than one thing wrong with this mock. Would be great if we would not have to do this at all. But for now that would mean that we have to copy the defaults to the project as well. Then again, do we really need to revert to the defaults each time we prepare?

Reply:
I think it is good to revert each time we prepare. User could accidentally delete a key from the file that is needed for the builds to work. Then we would like repopulat with the default.

It is also another way for users to quickly remove a change value back to the default without needing to remember what was the exact default value, as it would re-add the defaults.